test/src/main/java/org/apache/accumulo/test/functional/TabletManagementIteratorIT.java

```diff
@@ -163,14 +172,30 @@ public void test() throws AccumuloException, AccumuloSecurityException, TableExi
       assertEquals(1, findTabletsNeedingAttention(client, metaCopy2, tabletMgmtParams),
           "Only 1 of 2 tablets in table t1 should be returned");
 
+      // Remove location and set merge operation id on both tablets
+      // These tablets should not need attention as they have no WALs
+      setTabletHostingGoal(client, metaCopy4, t4, TabletHostingGoal.ALWAYS.name());
+      removeLocation(client, metaCopy4, t4);
+      setOperationId(client, metaCopy4, t4, null, TabletOperationType.MERGING);
```

Comment:
Could throw in a check before setting the merge op id.

```suggestion
      assertEquals(2, findTabletsNeedingAttention(client, metaCopy4, tabletMgmtParams),
              "Tablets have no location and a hosting goal of always, so they should need attention");
      setOperationId(client, metaCopy4, t4, null, TabletOperationType.MERGING);
```